index.html

```diff
@@ -1,13 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="index.css">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Books-Store</title>
+    <link rel="stylesheet" href="./index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1>Hello World</h1>
-  <script src="index.js"></script>
+    <header>
+        <nav>
+            <h3>awesome books</h3>
+            <ul>
+                <li><a href="#" id="list-link">Lists</a></li>
+                <li><a href="#" id="add-new-link">Add new</a> </li>
+                <li><a href="#" id="contact-link">Contact</a></li>
+            </ul>
+        </nav>
+        <div id="date"></div>
+    </header>
+
+    <section class="container" id="container">
+        <div id="lists">
+            <h1>All awesome Books</h1>
+            <div class= "display display-type-1" id="display"></div>
+        </div>
+       <div id="add-new">
+        <h1>Add a new book</h1>
+        <form id="add-new-form">
+            <input type="text" id="title" placeholder="title">
+            <input type="text" id="author" placeholder="author"> 
+            <div id="formbut"><button id="addbtn" onclick="a.addBooks()"> Add</button></div>
```

Comment:
- [ ] Please avoid inserting JavaScript function directly into your index.html file, Inserting JavaScript functions directly into the index.html file is not considered a good practice in modern web development. While it might work for small projects or simple experiments, it can quickly become unmanageable and difficult to maintain as your project grows in complexity. Please resolve it. :slightly_smiling_face: